### ISSUE TITLE
I need get the color name from hexcode

### ISSUE THREAD

**Issue by mohanarchu** [flutter/flutter#26943]
I got an color class from flutter developers, now i want to get the name by using list 
the class is https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/colors.dart

**Comment by Kartik1607** [comment]
Please refer to your original issue https://github.com/flutter/flutter/issues/26882 .

**Comment by mohanarchu** [comment]
> Please refer to your original issue #26882 .

can i convert hexcode to RGB?

**Comment by Kartik1607** [comment]
Yes, you can. It would be helpful if you try first and then if you are still stuck you can ask for help :)

**Comment by kamleshwebtech** [comment]
static const List<MaterialColor> primaries = <MaterialColor>[
    red,
    pink,
    purple,
    deepPurple,
    indigo,
    blue,
    lightBlue,
    cyan,
    teal,
    green,
    lightGreen,
    lime,
    yellow,
    amber,
    orange,
    deepOrange,
    brown,
    // The grey swatch is intentionally omitted because when picking a color
    // randomly from this list to colorize an application, picking grey suddenly
    // makes the app look disabled.
    blueGrey,
  ];

How can i get color name from this list, please suggest. Thanks.
